### PR TITLE
10 json files describing products for each set

### PR DESCRIPTION
**Pull request by aaronaskew**

### CODE REVIEW

File: CardShop/Repositories/Data/CardSets/tat.json
Comment (augenblick) [inline CardShop/Repositories/Data/CardSets/tat.json:1]: I think the dark side version of the "Boonta Eve Podrace" is also supposed to be R9 rarity (it's worded weirdly in the chonk, so feel free to double check).

Comment (augenblick) [reply CardShop/Repositories/Data/CardSets/tat.json:1]: image

Comment (aaronaskew) [reply CardShop/Repositories/Data/CardSets/tat.json:1]: ah thanks! i agree.

Comment (aaronaskew) [reply CardShop/Repositories/Data/CardSets/tat.json:1]: fixed in latest commit

**Review by augenblick — APPROVED** [review]
sweeeet

**Pull request merged**